library/src/main/res/values-v9/strings.xml

```diff
@@ -2,5 +2,5 @@
 <!-- Copyright 2012 Square, Inc. -->
 
 <resources>
-  <string name="month_name_format">LLLL yyyy</string>
+  <string name="month_name_format" translatable="false">LLLL yyyy</string>
```

Comment:
revert this: date format strings should be translatable so the keys can be reordered in different languages